conans/model/conanfile_interface.py

```diff
@@ -0,0 +1,11 @@
+
+
+class ConanFileInterface:
+
+    def __init__(self, conanfile):
+        self._conanfile = conanfile
+
+    @property
+    def new_env(self):
```

Comment:
new env?

Reply:
Yes, it is a placeholder for a better name. The problem is ``environment()`` is the method name, and ``self.env`` already exists in conanfile, to contain the aggregated EnvInfo.